src/pages/Admin/Review/index.tsx

```diff
@@ -1,22 +1,22 @@
 import { Route, Routes } from "react-router-dom";
+import { LinkGroup } from "../Sidebar/types";
 import { adminRoutes } from "constants/routes";
 import Layout from "../Layout";
 import Proposal from "../Proposal";
 import Proposals from "../Proposals";
+import { LINKS } from "../constants";
 import Applications from "./Applications";
-import Nav from "./Nav";
 import Templates from "./Templates";
 
+const LINK_GROUPS: LinkGroup[] = [
+  { links: [LINKS[adminRoutes.index]] },
```

Comment:
`admin/revew` index route `/` should be called `"Applications"` not `"Dasboard"`